src/test/java/org/elasticsearch/bwcompat/OldIndexBackwardsCompatibilityTests.java

```diff
@@ -139,54 +233,42 @@ void assertBasicSearchWorks() {
         ElasticsearchAssertions.assertNoFailures(searchReq.get());
```

Comment:
never mind, I see the sort now. Can we add a logger.info("--> testing basic search"), logger.info("--> testing basic search with sort")?